From 9812695da2de32f90ea47ba21e78e29437e4de8f Mon Sep 17 00:00:00 2001
From: John Levon <john.levon@joyent.com>
Date: Wed, 23 Oct 2019 17:46:27 +0000
Subject: [PATCH] OS-7598 Kernel needs to be built with retpolines (fix
 mismerge) Reviewed by: Jerry Jelinek <jerry.jelinek@joyent.com> Approved by:
 Jerry Jelinek <jerry.jelinek@joyent.com>

---
 usr/src/uts/intel/ia32/ml/modstubs.s | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/usr/src/uts/intel/ia32/ml/modstubs.s b/usr/src/uts/intel/ia32/ml/modstubs.s
index 8ea92d695a..3900b37d2c 100644
--- a/usr/src/uts/intel/ia32/ml/modstubs.s
+++ b/usr/src/uts/intel/ia32/ml/modstubs.s
@@ -237,10 +237,6 @@ fcnname/**/_info:							\
 	ret
 	SET_SIZE(stubs_common_code)
 
-#else
-#error "Trying to build modstubs on an unsupported arch"
-#endif	/* __amd64 */
-
 #define STUB(module, fcnname, retfcn)	\
     STUB_COMMON(module, fcnname, mod_hold_stub, retfcn, 0)
 
-- 
2.17.2 (Apple Git-113)

